frontend/pages/hosts/details/modals/CertificateDetailsModal/CertificateDetailsModal.tsx

```diff
@@ -168,7 +168,7 @@ const CertificateDetailsModal = ({
                     orientation="horizontal"
                   />
                 )}
-                {key_strength && (
+                {!!key_strength && (
```

Comment:
In this particular JSX idiom, literal zero value ends up getting displayed if not manually coerced to false